src/Components/WebAssembly/JSInterop/src/WebAssemblyJSObjectReference.cs

```diff
@@ -9,11 +9,14 @@ internal class WebAssemblyJSObjectReference : JSInProcessObjectReference, IJSUnm
     {
         private readonly WebAssemblyJSRuntime _jsRuntime;
 
-        public WebAssemblyJSObjectReference(WebAssemblyJSRuntime jsRuntime, long id) : base(jsRuntime, id)
+        public WebAssemblyJSObjectReference(WebAssemblyJSRuntime jsRuntime, long id)
+            : base(jsRuntime, id)
         {
             _jsRuntime = jsRuntime;
         }
 
+        internal new long Id => base.Id;
```

Comment:
Nitpick :)

If there's no product-code reason for this to exist, it's a bit weird for it to do so, and means that every time the class accesses its own `Id` it does through via an unnecessary extra method call. This suffix avoids that and makes the code more honest.

```suggestion
        internal long IdForTests => base.Id;
```

Reply:
Oh no wait, I see it's not only for tests.